### PR TITLE
prepare release v4.4.0

### PR DESCRIPTION
**Pull request by varshavaradarajan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## unreleased
 
+## v4.4.0 - 2022.11.02
+
+* Update CSI driver for Kubernetes 1.25
+  [[GH-461]](https://github.com/digitalocean/csi-digitalocean/pull/461)
```
Comment (timoreimann) [inline CHANGELOG.md:6]: Needs to say 464, not 461.

**Review by timoreimann — APPROVED** [review]
Feel free to merge once the Github reference has been updated.

File: CHANGELOG.md (hunk 1 above)
Comment (varshavaradarajan) [reply CHANGELOG.md:6]: 464 is this PR, 461 is the one that updates the k8s deps 1.25

Comment (timoreimann) [reply CHANGELOG.md:6]: Oh my bad, sorry.

**Pull request merged**